test/unit/spec/LoginRouter_spec.js

```diff
@@ -5,7 +5,7 @@ define([
   'backbone',
   'shared/util/Util',
   'util/CryptoUtil',
-  'util/Logger',
+  'shared/util/Logger',
```

Comment:
Since we are relying on Courage's logger, we should reference it directly.

Reply:
I wanted to make this a bit more clear in our tests that we are expecting a `console.warn` message from our model validation inside of Courage, as it only get passed up to a user/developer in a specific environment. Our new `Logger` class does **not** call this method.

Instead of removing/modifying the existing tests that make this assertion, do you have any recommendations?

Reply:
I think this is good. I would add a comment explaining this, so we know why we use Courage's Logger instead of the local.